pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -98,6 +98,38 @@ func Execute(configFile io.Reader) {
 		pruneAccessRecords,
 	)
 
+	// openshiftRouter groups the OpenShift related registry extensions
+	openshiftRouter := app.NewRoute().PathPrefix("/openshift/").Subrouter()
+
+	signatureRouter := openshiftRouter.NewRoute().PathPrefix("/signature/").Subrouter()
```

Comment:
Also I guess, we can use `handlers.NameRequired` flag in `app.RegisterRoute` in this case.